modules/ROOT/pages/csrf-security.adoc

```diff
@@ -46,7 +46,7 @@ The `X-Bonita-API-Token` is valid for the whole client session.
 Disabling the CSRF protection is not recommended as it will makes Bonita REST API unprotected from malicious attacks.
 
 But if you need to, the deactivation of the CSRF protection can be done in the file `security-config.properties` for the whole platform.
```

Comment:
the deactivation of the CSRF protection can be done in the file `security-config.properties` for the whole platform. => the CSRF protection can be deactivated in the file `security-config.properties` for the whole platform.

Reply:
I added this